givitsite/coordinate/admin.py

```diff
@@ -1,3 +1,8 @@
+import sys
 from django.contrib import admin
+from . models import CoordinatedItems
 
 # Register your models here.
```

Comment:
Is this comment still required? 
If so please write it like that: 
"# TODO: register the models here + issue number "

Reply:
Done